client.go

```diff
@@ -266,6 +269,9 @@ func (c *Client) Close() error {
 //
 // If no ProcessAt or ProcessIn options are provided, the task will be pending immediately.
 func (c *Client) Enqueue(task *Task, opts ...Option) (*TaskInfo, error) {
+	if task.Type() == "" {
```

Comment:
Like @crossworth suggested, let's `TrimSpace` here so that we can check for any blank strings.
Please make sure to update corresponding check in `servemux.go` as well!

Reply:
Added `TrimSpace` in `client.go` and `servemux.go`